src/concepts/components/properties.md

```diff
@@ -1,26 +1,551 @@
 ---
-description: Parent to child communication
+description: The component could expose property attributes to receive data from the parent html component
 ---
 
 # Properties
 
-Properties enable child and parent components to communicate with each other.
+It is a common need to receive data from the parent component using properties.
 
-## Derive macro
+The properties are defined in a dedicated Rust struct derivating `Properties` and `Clone`traits.
 
-Don't try to implement `Properties` yourself, derive it by using `#[derive(Properties)]` instead.
+> This struct is usually named `Props`
 
-{% hint style="info" %}
-Types for which you derive `Properties` must also implement `Clone`. This can be done by either using `#[derive(Properties, Clone)` or manually implementing `Clone` for your type.
-{% endhint %}
+The properties bag is attached to the state adding the field `props` in the component struct and defining the `Properties` `Type` in the component trait implementation.
 
-### Required attributes
+The properties have to be intialized in the `create` method. As for `link`, the framework push to the method the bag of the properties and can be used to attach it to the internal state.
```

Comment:
The properties aren't initialised in the `create` method, I think. Yew creates instances of the struct behind the scenes and passes them to `create` methods as needed.

Reply:
Make sense ... misunderstanding from my side. May be a mess as I was thinking mandatory the 'props' field in the state.

-> remove the sentence